docs/dev/setup/jenkins-gitlab.rst

```diff
@@ -63,7 +69,6 @@ the `Gitlab Server Quickstart <#gitlab-server-quickstart>`__ guide.
         user: artemis_admin
         password: artemis_admin
         url: http://localhost:8082
-        secret-push-token: AQAAABAAAAAg/aKNFWpF9m2Ust7VHDKJJJvLkntkaap2Ka3ZBhy5XjRd8s16vZhBz4fxzd4TH8Su # pre-generated or replaced in Automated Jenkins Server step 3
```

Comment:
Leaving out the `secret-push-token` defaults to the value `<token hash>` which results in a syntax error in Jenkins' `resources/templates/jenkins/config.xml`. Maybe remove the config section in `config.xml`.

Reply:
I would not edit the jenkins config for gitlab. This will be removed anyway.

Reply:
Moved to PR #9317.